docs/document/content/user-manual/shardingsphere-jdbc/builtin-algorithm/load-balance.cn.md

```diff
@@ -14,3 +14,15 @@ weight = 4
 类型：RANDOM
 
 可配置属性：无
+
+## 权重访问算法
+
+类型：WEIGHT
+
+可配置属性：
+
+> 使用中的读库都必须配置权重
+
+| *属性名称*                     | *数据类型* | *说明*                                         |
+| ------------------------------ | ---------- | ---------------------------------------------- |
+| \- <read-data_source-name> (+) | double     | 属性名字使用读库名字，参数填写读库对应的权重值  |
```

Comment:
Thank you!
I have a question, what should be the sum of all weight values, is there a limit to this? Need to be stated in the documentation?

Reply:
Ok, let me add